modules/tutorials/pages/java-tutorial/second-couchbase-application.adoc

```diff
@@ -0,0 +1,124 @@
+:page-pagination:
+
+= Your second Couchbase application: populating the course details
```

Comment:
Please put document title on the first line with the doc attributes below it.

Reply:
Can't right now. The page-pagination only seems to work reliably as the very first element on the page. Not sure why this is, because attributes don't work like that. I've dropped a note to OpenDevise to see if they can advise.